styles.css

```diff
@@ -607,12 +653,329 @@ section featured speakers
     z-index: 10;
   }
 
+  .header-content,
+  .descr,
+  .date,
+  .address {
+    width: 90%;
+    margin: 0 auto;
+  }
+
   .show-menu {
     display: block;
   }
 
   .main-program-link {
     display: none;
   }
+
+  .header-content,
+  .about-content {
+    max-width: 70%;
+    font-size: 4rem;
+    margin: 0 auto;
+  }
 }
 
+/*
+ ===================================
+****|| DESKTOP VERSION  MEDIA QUERIES ||****
+=================================== */
+@media screen and (min-width: 768px) {
+  .btn-toggle {
+    display: none;
+  }
+
+  .header {
+    position: relative;
+  }
+
+  .nav-bar {
+    position: sticky;
+  }
+
```

Comment:
[Optional] Kindly adjust the design to remove the horizontal scroll so as to improve the user experience.